aws_embedded_metrics/serializers/log_serializer.py

```diff
@@ -50,29 +52,39 @@ def create_body() -> Dict[str, Any]:
                 }
             return body
 
-        current_body: Dict[str, Any] = create_body()
+        current_body: Dict[str, Any] = {}
         event_batches: List[str] = []
         num_metrics_in_current_body = 0
+        missing_data = True
```

Comment:
Can you add some comments about what this is and consider thinking about a more intuitive name? It's not really "missing" data, something more like data left to process.

Reply:
I've added comments and renamed to "remaining_data".